client/src/components/tests/CKANPresentedOptionsField-test.js

```diff
@@ -50,43 +50,49 @@ const prepCKANApiMock = (options = null) => {
 };
 
 describe('CKANPresentedOptionsField', () => {
-  it('should render given select options', () => {
-    const props = { data: {
-        ...dataProps,
-        selectTypes: [{
-          value: SELECT_TYPE_ALL,
-          title: 'Option A',
-        }, {
-          value: SELECT_TYPE_CUSTOM,
-          title: 'Option B',
-        }]
-    } };
+  describe('render()', () => {
+    it('should render given select options', () => {
+      const props = {
+        data: {
+          ...dataProps,
+          selectTypes: [{
+            value: SELECT_TYPE_ALL,
+            title: 'Option A',
+          }, {
+            value: SELECT_TYPE_CUSTOM,
+            title: 'Option B',
+          }]
+        }
+      };
+
+      const component = shallow(
+        <CKANPresentedOptions LoadingComponent={LoadingComponent} {...props} />
+      );
 
-    const component = shallow(
-      <CKANPresentedOptions LoadingComponent={LoadingComponent} {...props} />
-    );
+      const options = component.find('.ckan-presented-options__option-group');
+      expect(options).toHaveLength(2);
+      expect(options.at(0).render().text()).toBe('Option A');
+      expect(options.at(1).render().text()).toBe('Option B');
+    });
 
-    const options = component.find('.ckan-presented-options__option-group');
-    expect(options).toHaveLength(2);
-    expect(options.at(0).render().text()).toBe('Option A');
-    expect(options.at(1).render().text()).toBe('Option B');
-  });
+    it('should allow specifying a default select type', () => {
+      const props = {
+        data: {
+          ...dataProps,
+          selectTypeDefault: SELECT_TYPE_CUSTOM,
+        }
+      };
 
-  it('should allow specifying a default select type', () => {
-    const props = { data: {
-        ...dataProps,
-        selectTypeDefault: SELECT_TYPE_CUSTOM,
-    } };
-
-    const component = shallow(
-      <CKANPresentedOptions LoadingComponent={LoadingComponent} {...props} />
-    );
+      const component = shallow(
+        <CKANPresentedOptions LoadingComponent={LoadingComponent} {...props} />
+      );
 
-    const options = component.find('.ckan-presented-options__option-group');
-    expect(options).toHaveLength(2);
-    const secondOption = options.at(1).render();
-    expect(secondOption.find('input').val()).toBe(SELECT_TYPE_CUSTOM);
-    expect(secondOption.find(':checked')).toHaveLength(1);
+      const options = component.find('.ckan-presented-options__option-group');
+      expect(options).toHaveLength(2);
+      const secondOption = options.at(1).render();
+      expect(secondOption.find('input').val()).toBe(SELECT_TYPE_CUSTOM);
+      expect(secondOption.find(':checked')).toHaveLength(1);
+    });
```

Comment:
These changes are all just indentation from wrapping it inside a `describe()` callback